webapp/hooks/index.ts

```diff
@@ -0,0 +1,5 @@
+export { default as useI18n } from './useI18n'
+export { default as useCountryIso } from './useCountryIso'
+export { default as usePrintView } from './usePrintView'
+export { default as useAppLoaded } from './useAppLoaded'
+export { default as useInitCountry } from './useInitCountry'
```

Comment:
aren't these hooks from the store?
I thought it might be better to have them under their own slice.
```webapp/store/app```

Reply:
Ok, I misunderstood previous comment